src/database/migrations/20190829010013-create-trip.js

```diff
@@ -0,0 +1,53 @@
+
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Trips', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      type: Sequelize.INTEGER,
+      primaryKey: true
+    },
+    tripId: {
+      type: Sequelize.STRING,
+      unique: true
+    },
+    userId: {
+      type: Sequelize.STRING,
+      references: {
+        model: 'Users',
+        key: 'userId'
+      }
+    },
+    departureDate: {
+      type: Sequelize.STRING,
+    },
+    returnDate: {
+      type: Sequelize.STRING,
+    },
+    tripType: {
+      type: Sequelize.STRING,
+    },
+    reason: {
+      type: Sequelize.TEXT,
+    },
+    currentOfficeLocation: {
+      type: Sequelize.STRING,
```

Comment:
We should save the currentOfficeLocation id here, not the name. so if and when we need the name of the currentOfficeLocation, we can retrieve it from the locations table using the ID